### DISCUSSION TITLE
Need of a video

### DISCUSSION THREAD

**Discussion by nemi4ko** [DaZombieKiller/FatePackageManager#5] [Q&A]
Could you please record a video of doing this repacking? I can't really understand how to use it even with a guide 😪

**Comment by DaZombieKiller** [comment]
![fpm](https://github.com/user-attachments/assets/d74937a9-ea4b-4397-87bd-16ba4f196061)
In this example, `boot_extracted` contains the files extracted from the original `boot.dat` like this:
![image](https://github.com/user-attachments/assets/460b2cf3-d584-4903-8392-c92d9f1f3163)
